### ISSUE TITLE
Issues with client/ server when closed

### ISSUE THREAD

**Issue by airhop** [thyer/CS340winter2016#682]
If you play the game for a few real turns, restart the server and finish your turn, itt breaks and won't continue so you restart the client and it bypasses the setup.

**Comment by thyer** [comment]
Wait, if you restart the server, doesn't it kill the existing game?

**Comment by cmriboldi** [comment]
It should kill the existing game. There is no way that the same game would persist without Phase 4 being done.

**Comment by thyer** [comment]
Yeah, that's my feeling too. @airhop were you expecting it to continue?

**Comment by KasiKarra** [comment]
From how the game played it appeared that the client didn't run through the finishturn during the setup rounds.  If that is true, we found a loophole that tricks the server into finishing the turn during the setup phase and start a game without settlements or roads and would be fixed with a simple if statement in the finishturn method.
We didn't expect the game to continue, but we didn't expect it to break down and die either.

**Comment by thyer** [comment]
Wait, so you're able to reboot up the server, issue commands, that that breaks the game on the new server? Is that correct?

**Comment by KasiKarra** [comment]
yes, I can drop by during your TA hours another time and show you or post a bunch of screenshots if either will help

**Comment by thyer** [comment]
Let's see if Team 1 can dupe this. If not, we may both need your help. Thanks for your offer!

**Comment by KasiKarra** [comment]
Deal  Here is a better steps for reproduction (sorry explanations are so hard on little sleep :) )

1. Log in players and play a few rounds.
2. Keeping the client screens up, restart the server
3. Finish the turn for the given player
4. Close the client screens
5. Log back in and begin your empty game
